docs/cloud/ui/interactive-api.md

```diff
@@ -1,6 +1,113 @@
 # Interactive API
 
 
-The Interactive API is an embedded GraphQL client that allows you to access all of your data with complete flexibility. Thanks to features like schema introspection and automatic authentication, the interactive API makes it simple to access Cloud. You can also see the GraphQL API documentation directly inline.
+The Interactive API is an embedded GraphQL client that allows you to access all of your data with complete flexibility. Thanks to features like schema introspection and automatic authentication, the interactive API makes it simple to access Cloud. You can also see the GraphQL API documentation directly inline.  
+
+![](/cloud/ui/interactive-api.png)
+
+
+### Example Query
+
+To find out information about your flows, you could use a flow query.  The basics for that query would be:
+
+```graphql
+query {
+    flow {
+        name
+        id
+    }
+}
+```
+You can also use the autocomplete to show what other fields you could query:
+
+![](/cloud/ui/dropdown-on-interactive-api.png)
+
+### Nested Query
+
+To get information about the tasks connected to your flows, you could use a nested query:
```

Comment:
```suggestion
Let's build on our earlier query to get information about the tasks associated with our flows. One way to accomplish this would be to use a nested query:
```